app/connectors/parsers/DeleteCISDeductionsHttpParser.scala

```diff
@@ -34,16 +34,16 @@ object DeleteCISDeductionsHttpParser extends DESParser {
         case NO_CONTENT => Right(())
         case BAD_REQUEST | NOT_FOUND =>
```

Comment:
There is a new `422` error code, that probably it should be handled in this `match`? Otherwise it will be handled as an internal server error.

<img width="760" alt="image" src="https://user-images.githubusercontent.com/108010791/211365119-96836850-76c9-48c0-8946-c8312f2f5dba.png">

Reply:
Fixed.